planet/src/main/webapp/js/itinerary.js

```diff
@@ -7,6 +7,20 @@ function closeForm() {
     document.getElementById('add-event').style.display = 'none';
 }
 
+function handleHotelChange() {
+    if (typeof(Storage) !== "undefined") {
+        sessionStorage.hotel = document.getElementById('hotel-address').value;
```

Comment:
use getItem and setItem to correspond to spec: https://www.w3schools.com/jsref/prop_win_sessionstorage.asp

Reply:
Done!